libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateAlliedProjectile.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** CreateAlliedProjectile
+*/
+
+#include "CreateAlliedProjectile.hpp"
+
+namespace ecs
+{
+    std::size_t createNewAlliedProjectile(
+        World &world, Entity &ally, const std::string uuid, unsigned short networkId)
+    {
+        Position pos = ally.getComponent<Position>();
+        Damage damage = ally.getComponent<Damage>();
+        Velocity velocity = ally.getComponent<Velocity>();
+
+        Entity &entity = world.addEntity()
+                             .addComponent<Position>(pos.x + 102, pos.y + 45)
+                             .addComponent<Velocity>(300, 0)
+                             .addComponent<Weight>(1)
+                             .addComponent<Size>(40, 40)
+                             .addComponent<LifeTime>(100)
```

Comment:
LIfetime is now at 3, not important bc waiting for the lifetime bug fix

Reply:
Done